Tests/LinguaTests/Infrastructure/DirectoryOperations/Mock/MockFileManager.swift

```diff
@@ -3,8 +3,8 @@ import Foundation
 
 final class MockFileManager: FileManager {
   let files: [String]
-  var shouldThrowErrorOnRemoveItem = false
-  var shouldThrowErrorOnCreateDirectory = false
+  var shouldThrowErrorOnRemoveItem: String?
```

Comment:
Let's also rename these two variables to something that indicates a string and not a bool, such as: `var onRemoveItemError: String?`